src/Box/Box.js

```diff
@@ -36,6 +36,10 @@ const Box = React.forwardRef(
       flexGrow,
       position,
       size,
+      height,
+      width,
+      xScrollable = false,
```

Comment:
Since this is the box component primitive - we should make sure the props are more explicit/low-level. So in this case, I think you should switch this to use 3 overflow props (`overflowX`, `overflowY`, `overflow`) that each accept one of 'hidden', 'visible', 'auto', and 'scroll'. 

Theres a component called `ScrollBox` that is for x-axis scrolling that could then be updated to use the new prop internally, but expose its prop API to something like `scrollX` and `scrollY` booleans. Then it can be used when you need x or y scrolling. Lot of ideas here if you wanna discuss more.

Reply:
I like that idea and I agree, having three overflow props with four different possible values gives more control to the user without getting too complicated. I'll update this as soon as I can